src/routes/api.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from fastapi import APIRouter, Request, status
+from fastapi.responses import JSONResponse, Response
+
+from .utils import error_response
+
+
+router = APIRouter()
+
+
+@router.post('/api',
```

Comment:
Now, front-end app sends requests to `/api/v1` url.
You should change route for compatibility.

Reply:
Will be patched soon with other possible updates.
```suggestion
@router.post('/api/v1',
```